specs/window_functions.toml

```diff
@@ -0,0 +1,27 @@
+[setup]
+statement_files = ["sql/uservisits.sql"]
+statements = [
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00001*' with (compression = 'gzip')",
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00002*' with (compression = 'gzip')",
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00003*' with (compression = 'gzip')",
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00004*' with (compression = 'gzip')",
+    "copy uservisits from 's3://crate.amplab/data/1node/uservisits/part-00005*' with (compression = 'gzip')",
+    "refresh table uservisits"
+]
+
+[[queries]]
```

Comment:
I think a case with higher concurrency would be interesting as well.

Instead I wonder if we should get rid of the different functions, to keep the benchmarks here focused on the window function execution engine - and not particular window functions (they could be better tested in isolation with a jmh benchmark)

Reply:
I agree the functions don't really matter. As we group functions by windows, I'd say it'd be interesting to also see a case where we have 2 different windows in a query